### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_HomePage_ScreenReader: Narrator not announces information present under heading "debug Adapter.." in scan mode.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#205]
## **Actual:**

After pressing Up/Down arrow key Narrator not announces information present under heading "Debug Adapter Protocol" in scan mode.

Same issue is not observed with Chrome+ NVDA.

## **Expected:**

All the information should be announces by screen reader on scan mode after pressing Up/Down arrow keys.

##  **Environment Details:**
Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10
Screenreader: Narrator

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol

## **Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol
2. Debug Adapter Protocol home screen will get open.
3. Start Narrator and navigate to information present under heading and verify the issue.

## **User Impact:**

Screen reader Users will not get all the information of screen.

## **Attachment:**
![#205](https://user-images.githubusercontent.com/60422426/133749303-6f1f64aa-32c1-49ac-a505-66384d9d5d66.gif)

“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag**:#A11yTCS;#DesktopWeb;#Win10;#WCAG1.3.1;#FTP;#A11yMAS;#ChromiumEdge;#Narrator;#A11ySev1;#P2_Visual Studio Code Clien_Sep2021;#Visual Studio Code Client;#AILimited;#Element:Text;#P2_WCP;

**Comment by rzhao271** [comment]
This is an upstream issue.

I can repro the issue with Narrator + Edge 95.0.1020.40
I cannot repro the issue with Narrator + Chrome Version 95.0.4638.69

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
I have verified the issue in the Environment:https://microsoft.github.io/debug-adapter-protocol,
 Issue is not reproducing, It is fixed. Added attachment for reference

![Screenshot (327)](https://user-images.githubusercontent.com/98821792/178406512-87b5b44b-8de5-410b-b316-ab129040190a.png)

**Comment by prabhudevu** [comment]
#closed;
